DataSetExplorer/ManovaTest/ManovaTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using CodeModel.CaDETModel.CodeItems;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace DataSetExplorer.ManovaTest
+{
+    class ManovaTest
+    {
+        private readonly string _manovaScriptFile = "../../../ManovaTest/manova_test.py";
+        private readonly string _pythonPath = "../../../ManovaTest/venv/Scripts/python.exe";
+        private string _annotatedInstancesFile;
+        private string _metrics;
+        private string _independentVariable;
+        
+        public void SetupTestArguments(string annotatedInstancesFile, List<CaDETMetric> metrics, string independentVariable)
+        {
+            _annotatedInstancesFile = annotatedInstancesFile;
+            _metrics = "";
+            foreach (var metric in metrics)
+            {
+                _metrics += metric + " + ";
+            }
+            _metrics = _metrics[0..^2];
```

Comment:
?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/ranges-indexes

In this example, I take from index 0 to length - 2.

Reply:
Another option is to do _metrics = string.join(" + ", metrics).